.travis.yml

```diff
@@ -4,8 +4,10 @@ notifications:
   email:
     recipients:
       - dev@rocketmq.apache.org
+    if: branch = develop OR branch = master
```

Comment:
Why changing this file?

Reply:
en,not my changed ,i check out this feature from master, this changed is in master

Reply:
Do you need me to pull a branch from develop and resubmit it without this change?